addons/account_edi_ubl_cii_tax_extension/views/account_tax_views.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_account_invoice_form_inherit" model="ir.ui.view">
+        <field name="name">account.tax.form.inherit</field>
+        <field name="model">account.tax</field>
+        <field name="inherit_id" ref="account.view_tax_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='country_id']" position="after">
+                <field name="ubl_cii_tax_category_code"/>
+                <field name="ubl_cii_tax_exemption_reason_code"/>
```

Comment:
A help message would be nice, I think:
* "The VAT category code used for electronic invoicing purposes."
* "The reason why the amount is exempted from VAT or why no VAT is being charged, used for electronic invoicing purposes."